source/rally_and_tempest.rst

```diff
@@ -0,0 +1,97 @@
+.. include:: vars.rst
+
+==========================================
+Verifying the Cloud with Rally and Tempest
+==========================================
+
+`Rally <https://docs.openstack.org/rally/latest/>`_ is a test framework,
+and `Tempest <https://docs.openstack.org/tempest/latest/>`_ is the
+OpenStack API test suite.  In this guide, Rally is used to run Tempest tests.
+
+Requirements
+------------
+
+OpenStack tests are run from a host with access to the OpenStack APIs
+and external network for instances.
+
+The following software environment is needed:
+
+* OpenStack admin credentials, eg public-openrc.sh from kayobe-config/etc/kolla
+* A virtualenv setup with python-openstackclient installed
+
+Setup Rally for a new user
+--------------------------
+
+A good directory hierarchy would be `~/rally/shakespeare/tempest-recipes`.
+This is assumed in this guide.
+
+Install Rally into the virtualenv:
+
+.. code-block:: shell
+
+   pip3 install git+https://github.com/openstack/rally-openstack.git \
```

Comment:
You could use `opendev.org` instead of `github.com`. Any reason not to install the last release from PyPI?